CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ endif()
 set(KTIKZ_VERSION "0.13.2")
 set(KTIKZ_USE_KTEXTEDITOR TRUE CACHE BOOL "Use KTextEditor framework")
 
+set(QT_MIN_VERSION "6.5.0")
+set(KF6_DEP_VERSION "6.0.0")
```

Comment:
This `KF6_DEP_VERSION` should be used in `find_package(ECM ...)` below as in the current file for `konsole`:
https://github.com/KDE/konsole/blob/a4dfe903cd920978828a91ecfce2c8055fc5fa15/CMakeLists.txt